.github/azure-devops/action/task.json

```diff
@@ -0,0 +1,337 @@
+        "migrate diff": "Migrate Diff",
+        "migrate down": "Migrate Down",
+        "migrate lint": "Migrate Lint",
+        "migrate push": "Migrate Push",
+        "migrate test": "Migrate Test",
+        "monitor schema": "Monitor Schema",
+        "schema apply": "Schema Apply",
+        "schema lint": "Schema Lint",
+        "schema plan approve": "Schema Plan Approve",
+        "schema plan": "Schema Plan",
+        "schema push": "Schema Push",
+        "schema test": "Schema Test"
+      }
+    },
+    {
+      "type": "string",
+      "label": "Working directory",
+      "helpMarkDown": "Atlas working directory, default is project root",
+      "name": "working_directory",
+      "visibleRule": "action == migrate apply || action == migrate autorebase || action == migrate diff || action == migrate down || action == migrate lint || action == migrate push || action == migrate test || action == schema apply || action == schema lint || action == schema plan || action == schema plan approve || action == schema push || action == schema test"
+    },
+    {
+      "type": "string",
+      "label": "Configuration",
+      "helpMarkDown": "The URL of the Atlas configuration file. By default, Atlas will look for a file\nnamed `atlas.hcl` in the current directory. For example, `file://config/atlas.hcl`.\nLearn more about [Atlas configuration files](https://atlasgo.io/atlas-schema/projects).\n",
+      "name": "config",
+      "visibleRule": "action == migrate apply || action == migrate diff || action == migrate down || action == migrate lint || action == migrate push || action == migrate test || action == monitor schema || action == schema apply || action == schema lint || action == schema plan || action == schema plan approve || action == schema push || action == schema test"
+    },
+    {
+      "type": "string",
+      "label": "Environment",
+      "helpMarkDown": "The environment to use from the Atlas configuration file.  For example, `dev`.",
+      "name": "env",
+      "visibleRule": "action == migrate apply || action == migrate diff || action == migrate down || action == migrate lint || action == migrate push || action == migrate test || action == monitor schema || action == schema apply || action == schema lint || action == schema plan || action == schema plan approve || action == schema push || action == schema test"
+    },
+    {
+      "type": "string",
+      "label": "Variables",
+      "helpMarkDown": "A JSON object containing variables to be used in the Atlas configuration file.\nFor example, `{\"var1\": \"value1\", \"var2\": \"value2\"}`.\n",
+      "name": "vars",
+      "visibleRule": "action == migrate apply || action == migrate diff || action == migrate down || action == migrate lint || action == migrate push || action == migrate test || action == schema apply || action == schema lint || action == schema plan || action == schema plan approve || action == schema push || action == schema test"
```

Comment:
These inputs currently generated by my custom script. It grab all inputs from all actions and merging them here.
My next plan is have a single-source-of-trust (a yaml file) then generate manifest for all platform.